android/java/org/chromium/chrome/browser/privacy/settings/BravePrivacySettings.java

```diff
@@ -15,7 +15,10 @@
 
 import org.chromium.base.BraveFeatureList;
 import org.chromium.base.ContextUtils;
+import org.chromium.base.Log;
```

Comment:
i don't think we use Log anywhere. so we need to remove unused imports

Reply:
updated